cmd/kubeadm/app/phases/upgrade/staticpods_test.go

```diff
@@ -570,6 +597,73 @@ func getTempDir(t *testing.T, name string) (string, func()) {
 	}
 }
 
+func getPathMgrAndCfg(t *testing.T, moveFileFunc func(string, string) error) (StaticPodPathManager, *kubeadmapi.InitConfiguration) {
+	pathMgr, err := NewFakeStaticPodPathManager(moveFileFunc)
+	if err != nil {
+		t.Fatalf("couldn't run NewFakeStaticPodPathManager: %v", err)
+	}
+
+	constants.KubernetesDir = pathMgr.(*fakeStaticPodPathManager).KubernetesDir()
```

Comment:
constants.KubernetesDir is used in controlplanephase.CreateInitStaticPodManifestFiles(), so I think we need to set it, and it will be clean as following:
defer pathMgr.CleanupDirs()

Reply:
It's only used for the command lines that are embedded in the static pod manifests there. This should not be relevant to these tests, but even if it is, we'll just have to add `manifestsDir` to `GetStaticPodSpecs`.
Let's simply remove this and see if everything is working without it.